internal/controller/datamovement_controller.go

```diff
@@ -736,9 +737,10 @@ func filterByNamespace(namespace string) predicate.Predicate {
 
 // SetupWithManager sets up the controller with the Manager.
 func (r *DataMovementReconciler) SetupWithManager(mgr ctrl.Manager) error {
+	maxReconciles := runtime.GOMAXPROCS(0)
 	return ctrl.NewControllerManagedBy(mgr).
 		For(&nnfv1alpha1.NnfDataMovement{}).
-		WithOptions(controller.Options{MaxConcurrentReconciles: 128}).
+		WithOptions(controller.Options{MaxConcurrentReconciles: maxReconciles}).
```

Comment:
GOMAXPROCS defaults to the number of CPUs.  How many are on a rabbit?
The `cmd.wait()` likely allows Go to pick up another go proc, if one is available.  So this controller can probably handle many more data movements than GOMAXPROCS would allow.

I wonder if this controller should even have a max setting.

Reply:
Do you mean that it should default to 1?

Reply:
hmm, no.  I hadn't realized that was the default.  It should be a high number.  Let's stay with 128 until we see a reason to change it.

Reply:
This does equal 128 on the rabbit nodes:

```shell
root@nnf-dm-worker-pctgf:~# ./maxprocs
maxReconciles: 128
root@nnf-dm-worker-pctgf:~# nproc
128
```

Here's the code for `./maxprocs`: 

```golang
package main

import (
	"fmt"
	"runtime"
)

func main() {
	maxReconciles := runtime.GOMAXPROCS(0)
	fmt.Printf("maxReconciles: %d\n", maxReconciles)
}
```